neuralop/models/resample.py

```diff
@@ -47,10 +47,10 @@ def resample(x, res_scale, axis, output_shape = None):
     else:
         new_size = output_shape
 
-    if len(axis) == 1:
-        return F.interpolate(x, size = new_size[0], mode = 'linear', align_corners = True)
-    if len(axis) == 2:
-        return F.interpolate(x, size = new_size, mode = 'bicubic', align_corners = True, antialias = True)
+#     if len(axis) == 1:
+#         return F.interpolate(x, size = new_size[0], mode = 'linear', align_corners = True)
+#     if len(axis) == 2:
+#         return F.interpolate(x, size = new_size, mode = 'bicubic', align_corners = True, antialias = True)
```

Comment:
Why did you comment these?

Reply:
Can we update to still use them, they are faster

Reply:
I can update this. But then the neural operator can only be used with PyTorch version later than 1.9.1. Because, " antialias" is not available in the previous distribution of PyTorch.

Reply:
Makes sense, let's open an issue to address this in a separate PR.